src/cascadia/TerminalApp/AppCommandlineArgs.cpp

```diff
@@ -337,6 +338,61 @@ void AppCommandlineArgs::_buildFocusTabParser()
     setupSubcommand(_focusTabShort);
 }
 
+// Method Description:
+// - Adds the `focus-tab` subcommand and related options to the commandline parser.
+// - Additionally adds the `ft` subcommand, which is just a shortened version of `focus-tab`
+// Arguments:
+// - <none>
+// Return Value:
+// - <none>
+void AppCommandlineArgs::_buildMoveFocusParser()
+{
+    _moveFocusCommand = _app.add_subcommand("move-focus", RS_A(L"CmdMoveFocusDesc"));
+    _moveFocusShort = _app.add_subcommand("mf", RS_A(L"CmdMFDesc"));

```

Comment:
@zadjii-msft - dumb question.. let's say we introduce "m" for minimized, and then I write -mf. What will the command line do: minimized focus or move focus? :smile:

Reply:
Presumably, `wt -mf sp ; mf left` will still work, because the `--minimizeFocus` arg will be an _option_, not a _subcommand_. I'd think that CLI11 should be able to differentiate between the `-mf` and `mf` args. If not, that would definitely be an upstream bug 😄